SPECS/polkit/polkit.spec

```diff
@@ -111,24 +112,34 @@ fi
 %{_datadir}/gettext/its/polkit.loc
 
 %changelog
-* Sat May 09 00:21:16 PST 2020 Nick Samson <nisamson@microsoft.com> - 0.116-3
-- Added %%license line automatically
+*   Thu Jun 03 2021 Jon Slobodzian <josloboe@microsoft.com> - 0.116-4
+-   Patch for CVE 2021-3560
+
+*   Sat May 09 00:21:16 PST 2020 Nick Samson <nisamson@microsoft.com> - 0.116-3
```

Comment:
Please also remove the time information from the date, since we're already updating this bit. I think I'm going to send a separate PR cleaning up all of these across our specs.